enterprise/api/v1/serializers.py

```diff
@@ -412,7 +412,7 @@ class Meta:
         model = models.EnterpriseCustomerReportingConfiguration
         fields = (
             'enterprise_customer', 'active', 'delivery_method', 'email', 'frequency', 'day_of_month', 'day_of_week',
-            'hour_of_day', 'initialization_vector', 'encrypted_password', 'sftp_hostname', 'sftp_port', 'sftp_username',
+            'hour_of_day', 'encrypted_password', 'sftp_hostname', 'sftp_port', 'sftp_username',
```

Comment:
i think you have to remove encrypted password as well, unless that is what the new password field will be populating going forward?

Reply:
Yes. The new field is called `encrypted_password` and it specified as such here in the serializer configuration.